docs/_templates/2020/sponsors-simplelist.rst

```diff
@@ -0,0 +1,9 @@
+{% for key in ('keystone', 'patron', 'publisher', 'second', 'first') %}
+    {% if key in data.sponsors and data.sponsors[key] %}
+
+      {% for sponsor in data.sponsors[key] %}
+        * `{{ sponsor.name | capitalize() }} <{{ sponsor.link }}>`__
```

Comment:
This is broken. We should add a `brand` key to the dict, if we want to display it properly. Many companies have branding that breaks this, eg. `GitHub` and `Write the Docs`.

Reply:
Yes and yes. I figured I'd get this up and fix it later.